ui-speedspacechart/src/components/utils.ts

```diff
@@ -23,23 +23,20 @@ export const getGraphOffsets = (width: number, height: number, declivities?: boo
 
 /**
  * /**
- * Given a store including a list of speed data, return the maxSpeed
- * @param store
+ * Given a list of speed data, return the maxSpeed
+ * @param speeds
  */
-export const maxSpeedValue = (store: Store) => {
-  const speeds = store.speeds;
-  return Math.max(...speeds.map(({ value }) => value));
-};
-
+export const maxSpeedValue = (speeds: LayerData<number>[]) =>
```

Comment:
So, what is the motivation for changing the type here?

Is it to make it more obvious that only the `speeds` field from the store is used, so that callers can adjust their dependencies accordingly? 

(Would be nice to record this kind of infra in the commit message :P )

Reply:
Yes! The drawing function depends on this function, which depended on a fully populated store. So to decouple the first from the store, the second also needed to be decoupled.

Also I don't see why a function that computes things on an array and only depends on it should have any argument beyond the array in the first place

Reply:
Makes sense!